pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -1,11 +1,16 @@
 package storagecluster
 
 import (
+	"testing"
+
 	nbv1 "github.com/noobaa/noobaa-operator/v2/pkg/apis/noobaa/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
-	"testing"
 
+<<<<<<< HEAD
 	snapapi "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+=======
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+>>>>>>> a72e7430... updates unit-tests to account for monitoring resources
```

Comment:
This is some accidentially commited merge conflict marker, I guess from a reworking of the branch.
I think we need both parts (and the order reversed to satisfy gofmt).

Also this patch should be squashed with the previous patch, otherwise it fails unit tests.
```suggestion
	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
	snapapi "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
```